f8a_worker/start.py

```diff
@@ -7,6 +7,52 @@
 from f8a_worker.setup_celery import init_celery, init_selinon
 import raven
 from raven.contrib.celery import register_signal, register_logger_signal
+from selinon import Dispatcher
+from datetime import datetime
+import json
+from selinon.errors import SelinonException
+
+
+class NotABugFatalTaskError(SelinonException):
```

Comment:
Can't this be imported from selinon itself?

`from selinon.errors import NotABugFatalTaskError`

Reply:
@arajkumar this Exception is not is Selinon,  in worker also we have created in f8a_worker/errors.py

Reply:
https://github.com/selinon/selinon/blob/master/selinon/errors.py
This Exception is not present in Selinon, a custom NotABugFatalTaskError is created because we are using same in workers as well and I think it is ignored by sentry in reporting.